projects/ion/src/lib/input/input.component.spec.ts

```diff
@@ -136,4 +136,45 @@ describe('InputComponent', () => {
       expect(mockFn).toHaveBeenLastCalledWith(value);
     });
   });
+
+  describe('Clear Button events', () => {
+    const mockFn = jest.fn();
+    const value = 'input-with-clear-button';
+
+    beforeEach(async () => {
+      await sut({
+        valueChange: { emit: mockFn } as SafeAny,
+        clearButton: true,
+      });
+    });
+
+    afterEach(async () => {
+      mockFn.mockClear();
+    });
+
+    it('should render the clear button when informed and input have value', async () => {
+      userEvent.type(screen.getByTestId('input-element'), value);
```

Comment:
```suggestion
      let input = screen.getByTestId('input-element');
      userEvent.type(input, value);
```
This is just an example, you can set these similar gets on screen in a variable and set this on beforeEach, since every test will use it.
You can do this to clear-button too.